vm/interpreter/src/default_runtime.rs

```diff
@@ -660,7 +661,7 @@ where
             .on_method_invocation(msg.value(), msg.method_num()),
     )?;
 
-    if msg.value() > &TokenAmount::from(0) {
+    if !msg.value().is_zero() {
```

Comment:
Wait, why can this be negative?

Reply:
That is handled in the transfer function

Reply:
oh yeah, that's super weird, was wondering about that failed test case